src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse2.cs

```diff
@@ -14,858 +14,870 @@ namespace System.Runtime.Intrinsics.X86
         /// </summary>
         public static Vector128<byte> SetVector128(byte e15, byte e14, byte e13, byte e12, byte e11, byte e10, byte e9, byte e8, byte e7, byte e6, byte e5, byte e4, byte e3, byte e2, byte e1, byte e0)  => SetVector128(e15, e14, e13, e12, e11, e10, e9, e8, e7, e6, e5, e4, e3, e2, e1, e0);
         /// <summary>
-        /// __m128i _mm_set_epi16 (short e7, short e6, short e5, short e4, short e3, short e2, short e1, short e0)
+        /// __m128i _mm_set_epi16 (short e7, short e6, short e5, short e4, short e3, short e2, short e1, short e0); HELPER
         /// </summary>
         public static Vector128<short> SetVector128(short e7, short e6, short e5, short e4, short e3, short e2, short e1, short e0) => SetVector128(e7, e6, e5, e4, e3, e2, e1, e0);
         /// <summary>
-        /// __m128i _mm_set_epi16 (short e7, short e6, short e5, short e4, short e3, short e2, short e1, short e0)
+        /// __m128i _mm_set_epi16 (short e7, short e6, short e5, short e4, short e3, short e2, short e1, short e0); HELPER
         /// </summary>
         public static Vector128<ushort> SetVector128(ushort e7, ushort e6, ushort e5, ushort e4, ushort e3, ushort e2, ushort e1, ushort e0) => SetVector128(e7, e6, e5, e4, e3, e2, e1, e0);
         /// <summary>
-        /// __m128i _mm_set_epi32 (int e3, int e2, int e1, int e0)
+        /// __m128i _mm_set_epi32 (int e3, int e2, int e1, int e0); HELPER
         /// </summary>
         public static Vector128<int> SetVector128(int e3, int e2, int e1, int e0) => SetVector128(e3, e2, e1, e0);
         /// <summary>
-        /// __m128i _mm_set_epi32 (int e3, int e2, int e1, int e0)
+        /// __m128i _mm_set_epi32 (int e3, int e2, int e1, int e0); HELPER
         /// </summary>
         public static Vector128<uint> SetVector128(uint e3, uint e2, uint e1, uint e0) => SetVector128(e3, e2, e1, e0);
         /// <summary>
-        /// __m128i _mm_set_epi64x (__int64 e1, __int64 e0)
+        /// __m128i _mm_set_epi64x (__int64 e1, __int64 e0); HELPER
         /// </summary>
         public static Vector128<long> SetVector128(long e1, long e0) => SetVector128(e1, e0);
         /// <summary>
-        /// __m128i _mm_set_epi64x (__int64 e1, __int64 e0)
+        /// __m128i _mm_set_epi64x (__int64 e1, __int64 e0); HELPER
         /// </summary>
         public static Vector128<ulong> SetVector128(ulong e1, ulong e0) => SetVector128(e1, e0);
         /// <summary>
-        /// __m128d _mm_set_pd (double e1, double e0)
+        /// __m128d _mm_set_pd (double e1, double e0); HELPER
         /// </summary>
         public static Vector128<double> SetVector128(double e1, double e0) => SetVector128(e1, e0);
 
         /// <summary>
-        /// __m128d _mm_set_sd (double a)
+        /// __m128d _mm_set_sd (double a); HELPER
         /// </summary>
-        public static Vector128<double> SetScalar(double value) => SetScalar(value);
```

Comment:
Named changed to `SetScalarVector128`

Reply:
IMO from user perspective it is better to have names with Vector128/Vector256 even when Vector128 is implied.